Files/Views/ModernShellPage.xaml

```diff
@@ -244,17 +315,38 @@
                     Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
                     Navigated="ItemDisplayFrame_Navigated" />
 
+                <Custom:GridSplitter
+                    Grid.Column="1" Grid.Row="1"
+                    ResizeBehavior="BasedOnAlignment"
+                    Background="Transparent"
+                    x:Name="PreviewPaneGridSplitter" x:Load="{x:Bind InstanceViewModel.PreviewPaneEnabled, Mode=OneWay}">
+                    <Custom:GridSplitter.Element>
+                        <Grid>
+                            <TextBlock HorizontalAlignment="Center" IsHitTestVisible="False" VerticalAlignment="Center" x:Name="GripperGlyphTextBlock"
+                               Text="" FontSize="36" Foreground="{StaticResource SystemChromeHighColor}" FontFamily="{StaticResource FluentUIGlyphs}">
+                            </TextBlock>
+                        </Grid>
+                    </Custom:GridSplitter.Element>
+                </Custom:GridSplitter>
+
+                <controls:PreviewPane Grid.Row="1" Grid.Column="2" x:Name="PreviewPane" 
+                                    SelectedItems="{x:Bind ContentPage.SelectedItems, Mode=OneWay}"
```

Comment:
`BaseLayout` contains SelectedItem property, so what about to use it?

Reply:
The SelectedItem property doesn't update after the selection changes.

Reply:
It does.
![image](https://user-images.githubusercontent.com/20501502/105517927-51383580-5ce0-11eb-93c5-eb4927f6214b.png)

Reply:
Odd. I had tried using it at first, but I switched to SelectedItems because it wasn't updating.
I guess I'll change it back.

Reply:
Optionally you can add `NotifyPropertyChanged(nameof(SelectedItem));` inside.

Reply:
@tsvietOK SelectedItem will always show the first item in the list, no matter how many items are selected. This will not work with the PreviewPane, as it should not show anything when multiple items are selected. 

![image](https://user-images.githubusercontent.com/59544401/105520219-43a68f80-5c8f-11eb-9a72-604422f085b2.png)